packages/react-ui/.out/iframe.html

```diff
@@ -0,0 +1,75 @@
+<!doctype html><html lang="en"><head><meta charset="utf-8"><title>Storybook</title><meta name="viewport" content="width=device-width,initial-scale=1"><base target="_parent"><style>:not(.sb-show-main) > .sb-main,
```

Comment:
Does `.out` need to be versionned ?

Reply:
Yes, netlify will watch this folder and update the static website everytime new files are pushed

Reply:
Netlify Can also build so we dont need to version the build (see emjpm-ui)

Reply:
Cool, I haven't really looked at it yet. ! I'm waiting to finish the migration